holopy/core/metadata.py

```diff
@@ -98,6 +101,14 @@ def wavelen_in(self, medium_index):
         return self.wavelen/medium_index
 
     @property
+    def polarization(self):
+        return self._polarization
+
+    @polarization.setter
+    def polarization(self, val):
+        self._polarization = np.array(val)
+
```

Comment:
What is the logic behind using a property/setter here rather than just a simple variable?

You have different behavior with None in the constructor and setter, you should probably move the None check into the setter and then use the setter in the constructor (unless setters don't work in constructors, I can't remember for sure)